src/routes/v2/admin.rs

```diff
@@ -319,3 +321,13 @@ pub async fn trolley_webhook(
 
     Ok(HttpResponse::NoContent().finish())
 }
+
+#[post("/_force_reindex")]
```

Comment:
*theoretically* this should be safe behind an admin wall, but since its only really used for testing anyway, I should probably just put it behind a cfg test wall.